docs/AppService/OnAppServiceData/index.md

```diff
@@ -11,8 +11,8 @@ Purpose
 
 !!! note
 
-* **HMI->SDL** if HMI App Service Provider (ASP) is sending updated service data  
-* **SDL->HMI** if HMI App Service Consumer (ASC) is receiving updated service data
+* **HMI->SDL** if HMI App Service Provider (<span title="App Service Provider">ASP</span>) is sending updated service data  
+* **SDL->HMI** if HMI App Service Consumer (<span title="App Service Consumer">ASC</span>) is receiving updated service data
```

Comment:
Do you need to have the span here since you preface the abbreviation with the full name?

Reply:
Removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/146/commits/d71dde80f9c28f9452c70a2f5b12d551a9505129